formular.html

```diff
@@ -10,25 +10,13 @@ <h1>Date Matcher</h1>
     </header>
      
     <form>
-     <fieldset>
-      <ul>
-       <li>
-        <p>
-        <label for="Date1"></label>
-        <input type="date" name="Date1" id="Date1">
-        </p>
-       </li>
-       <li>
-        <p>
-        <label for="Date2"></label>
-        <input type="date" name="Date2" id="Date2">
-        </p>
-       </li>
-      </ul>
-     </fieldset>
-      <p>
+      <label for="Date1"></label>
```

Comment:
There's a technique to put an `unput` tag *inside* its `label` tag. That way we organically tell what the label labels and can drop IDs

Reply:
Do I have to change it with this technique? I didn't understand, is it true or I have to do changes and send to you?
And what is this technique?

Reply:
1. Your labels are empty, the contain no text.
2. The technique without id is:
```
<label>
    Date 1: 
    <input type="date" name="date1">
</label>
```

this way a label is connected with an input organically, without *for* and *id*